core/src/test/scala/unit/kafka/coordinator/transaction/TransactionMarkerRequestCompletionHandlerTest.scala

```diff
@@ -187,6 +187,12 @@ class TransactionMarkerRequestCompletionHandlerTest {
     verifyRetriesPartitionOnError(Errors.NOT_ENOUGH_REPLICAS_AFTER_APPEND)
   }
 
+  @Test
```

Comment:
@guozhangwang there are currently 0 unit-tests for this, just like most of `KafkaApis`. I've been trying to write a unit test for it, but it isn't as simple as just mocking the `ReplicaManager`